Apps/Sandcastle/gallery/3D Tiles.html

```diff
@@ -58,11 +58,11 @@
     }));
 
     return Cesium.when(tileset.readyPromise).then(function(tileset) {
-        var boundingBox = tileset.boundingVolume;
-        var center = Cesium.Rectangle.center(boundingBox.rectangle);
-        var height = boundingBox.maximumHeight + 500.0;
+        var boundingSphere = tileset.boundingSphere;
```

Comment:
In a new PR, can you see if we can replace this with [viewer.camera.viewBoundingSphere](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/Camera.js#L2603-L2616)?